src/software/Localization/voctreeQueryUtility.cpp

```diff
@@ -3,14 +3,25 @@
 #include <openMVG/voctree/databaseIO.hpp>
 #include <openMVG/voctree/vocabulary_tree.hpp>
 #include <openMVG/voctree/descriptor_loader.hpp>
+#include <openMVG/matching/indMatch.hpp>
 #include <openMVG/logger.hpp>
+#include <openMVG/types.hpp>
+#include <openMVG/voctree/databaseIO.hpp>
+#include <openMVG/sfm/sfm_data.hpp>
+#include <openMVG/sfm/sfm_data_io.hpp>
+#include <openMVG/sfm/pipelines/sfm_engine.hpp>
+#include <openMVG/sfm/pipelines/sfm_features_provider.hpp>
+#include <openMVG/sfm/pipelines/sfm_regions_provider.hpp>
+
+#include <Eigen/Core>
+
+//#include "selection.hpp"
+#include <opencv2/opencv.hpp>
```

Comment:
@fabiencastan @jmelou both to be removed
opencv is not used (and if it were there should be some conditional compiling here or in the CMakeLists as opencv is not a required dependency)